cluster/images/conformance/Makefile

```diff
@@ -15,14 +15,18 @@
 # Build the conformance image.
 #
 # Usage:
-#   [ARCH=amd64] [REGISTRY="staging-k8s.gcr.io"] make (build|push) VERSION={some_released_version_of_kubernetes}
+#   [ARCH=amd64] [REGISTRY="k8s.gcr.io"] make (build|push) VERSION={some_released_version_of_kubernetes}
 
-REGISTRY?=staging-k8s.gcr.io
+REGISTRY?=k8s.gcr.io
```

Comment:
this should have remained staging - you can't push to k8s.gcr.io directly, only to staging.

Reply:
aha. will fix in a next rev